src/Analyzers/CSharp/Analyzers/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionDiagnosticAnalyzer.cs

```diff
@@ -48,7 +55,7 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
 
             var options = context.Options;
             var cancellationToken = context.CancellationToken;
-            var optionSet = options.GetDocumentOptionSetAsync(syntaxTree, cancellationToken).GetAwaiter().GetResult();
+            var optionSet = options.GetOptions(syntaxTree, cancellationToken);
```

Comment:
can the GetOptoins change not be broken out?

Reply:
Yes, this is being done by @JoeRobich's PR: https://github.com/dotnet/roslyn/pull/38675. If his change goes in first, you will not see these changes in my PR.